Makefile.js

```diff
@@ -695,7 +706,9 @@ target.gensite = function(prereleaseVersion) {
 
                 text = `${ruleHeading}${isRecommended ? RECOMMENDED_TEXT : ""}${isFixable ? FIXABLE_TEXT : ""}\n${ruleDocsContent}`;
 
-                text = `---\ntitle: ${ruleName} - Rules\nlayout: doc\n---\n<!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->\n\n${text}`;
+                title = `${ruleName} - Rules`;
+
+                editLink = `https://github.com/eslint/eslint/edit/master/docs/rules/${baseName}`;
```

Comment:
Nitpick: Is it necessary to assign `editLink` separately when the page is in the `rules` directory? Wouldn't `` `https://github.com/eslint/eslint/edit/master/${filePath}` `` work in both cases?

Reply:
Awesome! I didn’t notice that.